spyder/plugins/explorer/widgets.py

```diff
@@ -859,7 +883,183 @@ def create_func(fname):
 
     def go_to_parent_directory(self):
         pass
-        
+
+    def copy_path(self, fnames=None, method="absolute"):
+        """Copy absolute or relative path to given file(s)/folders(s)."""
+        cb = QApplication.clipboard()
+        explorer_dir = self.fsmodel.rootPath()
+        explorer_dir = ('os.sep'.join(explorer_dir.strip('os.sep').
+                                      split('os.sep')[1:]))
+        if fnames is None:
+            fnames = self.get_selected_filenames()
+        if not isinstance(fnames, (tuple, list)):
+            fnames = [fnames]
+        fnames = [_fn.replace(os.sep, "/") for _fn in fnames]
+        if len(fnames) > 1:
+            if method == "absolute":
+                clipboard_files = ',\n'.join('"' + _fn + '"' for _fn in fnames)

```

Comment:
Per bcolson's comments elsewhere, you should use the file's EOL char instead of the hardcoding LF here and below.

Reply:
How I can get this fun `get_line_separator()`?? using `eol_chars = self.get_line_separator()` gives me attribute error directly.

Reply:
Actually, this seemed to work anyway in my testing. @bcolsen ?

Reply:
@Khalilsqu You would have to get the current Editor. However, you actually don't need to do this at all since the Editor's ``paste()`` already does this for you and I tested it and it works fine; I don't know what I was thinking here. Sorry!